src/components/MyKiva/JournalUpdateCard.vue

```diff
@@ -199,7 +199,7 @@ onMounted(() => {
 
 const useFunds = () => {
 	$kvTrackEvent('portfolio', 'click', 'repayment-update-read-more', update.value.id);
-	window.location.href = '/lend/filter';
+	window.location.href = `/lend/filter?header=You-can-impact-${update.value.livesImpacted}-lives-right-now`;
```

Comment:
Per ticket, this value actually needs to be `Where X = the user’s current balance divided by $25 + 1`